exercises/pangram/test/Tests.hs

```diff
@@ -0,0 +1,64 @@
+{-# LANGUAGE RecordWildCards #-}
+
+import Data.Foldable     (for_)
+import Test.Hspec        (Spec, describe, it, shouldBe)
+import Test.Hspec.Runner (configFastFail, defaultConfig, hspecWith)
+
+import Pangram (isPangram)
+
+main :: IO ()
+main = hspecWith defaultConfig {configFastFail = True} specs
+
+specs :: Spec
+specs = describe "pangram" $
+          describe "isPangram" $ for_ cases test
+  where
+    test Case{..} = it description $ isPangram input `shouldBe` expected
+
+-- Test cases adapted from
+-- `exercism/x-common/exercises/pangram/canonical-data.json`
+-- on 2017-03-28.
```

Comment:
I'm not sure if the structure I used here was explicitly made standard/convention or simply accepted by adapting from other tests (I simply followed/copied from other tests).

That said, now that we have more data to include, I'm going to propose the following:

```
Adapted from
Source: exercism/x-common/exercises/pangram/canonical-data.json
Version: 1.0.0
Date: 2017-03-28
```

`Test cases` seems redundant since that's what the file is about.